Cargo.toml

```diff
@@ -21,22 +21,22 @@ name = "amethyst"
 path = "src/main.rs"
 
 [dependencies]
-ansi_term = "0.9"
-clap = "2"
-env_proxy = "0.2"
-error-chain = "0.11"
-lazy_static = "1.0.0"
-liquid = "0.14.3"
-regex = "0.2.10"
-reqwest = "0.7.1"
-serde = "1.0"
-serde_derive = "1.0"
-serde_json = "1.0"
-walkdir = "2.1.4"
+ansi_term = "0.11.0"
+clap = "2.32.0"
+env_proxy = "0.2.0"
+error-chain = "0.12.0"
```

Comment:
Any specific reason to force patch versions?

Reply:
They're not forced?

Reply:
To be frank, I just used `cargo upgrade`, provided by [killercup's cargo-edit](https://github.com/killercup/cargo-edit). If you leave the versions like that, Cargo interprets them as caret requirements ([source](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio)).